packages/react-components/src/components/utils/longPress.ts

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+// Number of milliseconds that indicates a press was a 'long press'.
+// 400ms is the current Android long press time: https://android.googlesource.com/platform/packages/apps/Settings.git/+/06dfd7566/res/values/arrays.xml#731
+// 500ms is the current iOS long press time: https://developer.apple.com/documentation/uikit/uilongpressgesturerecognizer/1616423-minimumpressduration
+// However both iOS and Android allow customization of this behavior.
+// We will default to 500ms and gather feedback on this value.
+const LONG_PRESS_TIMEOUT_MS = 500;
+
+// Cancel the long press on movement. If this is too aggressive this should be updated to detect
+// if movement goes out of bounds of the element instead of cancelling on any movement.
+const CANCEL_ON_MOVEMENT = true;
+
+// Singleton that tracks whether there is an active touchstart event.
+let longPressTimerHandle: number | undefined = undefined;
```

Comment:
Don't use globals.

This should be a `use*` function, so that you can use `useRef` instead.

Similar to what the `use-long-press` package does (if we decide to not use that package for some reason).

Reply:
removed, this is all handled by use-long-press